test/common/index.js

```diff
@@ -246,7 +246,7 @@ Object.defineProperty(exports, 'opensslCli', {get: function() {
 
 Object.defineProperty(exports, 'hasCrypto', {
   get: function() {
-    return process.versions.openssl ? true : false;
+    return Boolean(process.versions.openssl);
```

Comment:
Not a huge fan of `!!` outside of minimal code competitions.
Willing to change if anyone has a strong opinion.